easybuild/easyconfigs/s/Shannon/Shannon-30a861-intel-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'Tarball'
+
+name = 'Shannon'
+version = '30a861'
```

Comment:
use `datestamp` as a version, and rename sources on download (see http://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#alternative-formats-for-sources)